src/EscapeHit.Invoice/Commands/CreateInvoice/CreateInvoiceCommandMapping.cs

```diff
@@ -12,9 +12,19 @@ public class CreateInvoiceCommandMapping : BaseMapping<InvoiceEntity, CreateInvo
     {
         public CreateInvoiceCommandMapping()
         {
+            //todo: implement declarative mapping
             Field(e => e.CustomerNumber).From(e => e.CustomerNumber);
```

Comment:
@viktor-kulkov I assume these settings must have persisted and been handled somewhere deeper in the framework. But it is not happening now. This is why I went ahead and introduced explicit mapping for now.